draft-ietf-quic-http.md

```diff
@@ -693,6 +708,12 @@ Push ID:
 Header Block:
 : HPACK-compressed request headers for the promised response.
 
+A server MUST NOT use a Push ID that is larger than the client has provided in a
+MAX_PUSH_ID frame ({{frame-max-push-id}}) or the SETTINGS_MAX_PUSH_ID setting
+({{settings-parameters}}).  A client that receives a PUSH_PROMISE that contains
+a larger Push ID than is allowed MUST be treated as a connection error of type
+HTTP_MALFORMED_PUSH_PROMISE.
```

Comment:
"is allowed" makes it sound like an absolute limit; might be worth rewording slightly.